deepgrp/__main__.py

```diff
@@ -310,16 +310,43 @@ def train(args: argparse.Namespace, options: dgmodel.Options) -> None:
         parameter.fromdict(options.todict())
         logdir = args.logdir
 
-        # add _LOG.xxx functions for logging
-        # include a check if preprocessed?
+        # get which chromosome is used for training and which for validation
+        if args.trainfile[-10:] != '.fa.gz.npz' or \
+            args.validfile[-10:] != '.fa.gz.npz':
+                # train or valid file is not valid
+                sys.stderr.write('Training and validation file given must be ' +
+                                'in the format chr[chrnumber].fa.gz.npz, ' +
+                                'e.g. chr1.fa.gz.npz\n')
+                exit(1)
+        train_chr = os.path.basename(args.trainfile).split('.')[0]
+        val_chr = os.path.basename(args.validfile).split('.')[0]
+
         # check if logdir exists?
-        # how to load data in?
-        # specific name for trained model?
-        # load data into np.ndarray
-        _LOG.info("Loading %s as training data", args.trainfile)
-        train_data = np.load(args.trainfile, allow_pickle=False)
-        _LOG.info("Loading %s as validation data", args.validfile)
-        val_data = np.load(args.valdata, allow_pickle=False)
+        if not os.path.isdir(logdir):
+            # logdir is not valid
+            sys.stderr.write('Given logdir is not a Directory\n')
+            exit(1)
```

Comment:
I think it's fine to just create the directory if it not exists.We don't need to exit here, but maybe we can add some warning that the directory is going to be created, like `_LOG.warning(....)`